### ISSUE TITLE
Why is it impossible to find this extension by searching the Google Web Store?

### ISSUE THREAD

**Issue by mercmobily** [GoogleChromeLabs/text-app#563]
Apologies if this is an obvious question. However, I do feel this might be issue-worthy.
I find it absolutely impossibly to find this great, well maintained, well reviewed and well used editor ANYWHERE in the Google Web Store.

![image](https://user-images.githubusercontent.com/2128734/172643931-c2a1d809-66d0-4bce-88b8-2e61cfbf2f56.png)

Is there a specific reason for this? Am I missing something glaringly obvious?
I looked for _many_ pages... and no joy.

I also do wish there was a way to filter, when searching the store, by extensions that are meant to be used like this one, opening a new window and launched... But still.

Any ideas?

**Comment by BroJac5246** [comment]
I believe this is intentional, as the app comes pre-installed on Chromebooks, meaning there is no use for the webstore page. You don't want Windows and Mac users to stumble upon this app meant for Chromebooks.
However, should a Chromebook user need it, it is linked on `chrome://extensions/?id=mmfbcljfglbokpmkimbfghdkjmjhdgbg`

**Comment by mercmobily** [comment]
> You don't want Windows and Mac users to stumble upon this app meant for Chromebooks.

Why not? It's a great app!

**Comment by BroJac5246** [comment]
> > You don't want Windows and Mac users to stumble upon this app meant for Chromebooks.
> 
> Why not? It's a great app!

True, but remember this app is built by Google for Chromebooks, so it really isn't intended for anything else.

It might be possible to download it with the URL (I don't know), but, overall, this is just a built-in app for Chromebooks.
Also, Windows has Notepad, and I'm sure Mac has its own text editor, so this app wouldn't even be needed by the majority of users.
Also, it's a relatively simple app, which could be another factor- Chromebooks are known for their "lite" approach to things, which is definitely _not_ what Windows and Mac are known for.
